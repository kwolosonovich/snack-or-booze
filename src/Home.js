import React from "react";
import { Card, CardBody, CardTitle } from "reactstrap";

function Home() {

  return (
    <section className="col-md-8">
      <Card>
        <CardBody className="text-center">
          <CardTitle>
            <div className="font-weight-bold">
              Welcome to Silicon Valley's premier dive cafe!
            </div>
            <div >Cat</div>
          </CardTitle>
        </CardBody>
      </Card>
    </section>
  );
}

export default Home;
