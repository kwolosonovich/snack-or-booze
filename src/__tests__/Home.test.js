import React from "react";
import { render } from "@testing-library/react";
import App from "../App";
import { BrowserRouter } from "react-router-dom";
import Home from "../Home"
import { Card, CardBody, CardTitle } from "reactstrap";


test("renders wihtout crashing", function () {
  render(<Home />);
});

test("match snapshot", function () {
  const { asFragment } = render(
    <div className="font-weight-bold">
      Welcome to Silicon Valley's premier dive cafe!
    </div>
  );
  expect(asFragment()).toMatchSnapshot();
});
