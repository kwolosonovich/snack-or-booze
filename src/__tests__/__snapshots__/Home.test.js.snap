// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`match snapshot 1`] = `
<DocumentFragment>
  <div
    class="font-weight-bold"
  >
    Welcome to Silicon Valley's premier dive cafe!
  </div>
</DocumentFragment>
`;
